test/Generics/function_decls.swift

```diff
@@ -44,7 +44,7 @@ func f14<T : P>(x: T) -> T.C<Int> {}
 
 func f15<T : P>(x: T) -> T.D<Int> {}
 
-func f16<T : P>(x: T) -> T.D {} // expected-error {{reference to generic type 'T.D' (aka 'G') requires arguments in <...>}}
+func f16<T : P>(x: T) -> T.D {} // expected-error {{reference to generic type 'T.D' requires arguments in <...>}}
```

Comment:
Perhaps these aliases still deserve an 'aka' so that the source of generics is immediately clear to the user.

Reply:
Yeah, I’ll look into that. The reason we don’t do that now is because generic typealiases use an UnboundGenericType here instead of TypeAliasType, which is also weird.

Reply:
Oh, I see. On the other hand, it would be dangerous to allow something like `typealias Foo<T> = T` to be casually desugared before being fully bound.